tuner.c

```diff
@@ -50,13 +50,17 @@ cSatipTuner::cSatipTuner(cSatipDeviceIf &deviceP, unsigned int packetLenP)
   debug1("%s (, %d) [device %d]", __PRETTY_FUNCTION__, packetLenP, deviceIdM);
 
   // Open sockets
-  int i = 100;
+  int i = SatipConfig.GetPortRangeStart() ? SatipConfig.GetPortRangeStop() - SatipConfig.GetPortRangeStart() - 1 : 100;
+  int port = SatipConfig.GetPortRangeStart();
   while (i-- > 0) {
-        if (rtpM.Open(0) && rtcpM.Open(rtpM.Port() + 1))
-           break;
-        rtpM.Close();
-        rtcpM.Close();
-        }
+    if (rtpM.Open(port) && rtcpM.Open(rtpM.Port() + 1))
+       break;
+    rtpM.Close();
+    rtcpM.Close();
+    if (SatipConfig.GetPortRangeStart()) {
+      ++port;
+      }
```

Comment:
No need to use parenthesis in these one liners. Use a correct indentation for "if" blocks.